samples/ingestion/ingestion-client/StartTranscriptionByTimer/Config/AppConfig.cs

```diff
@@ -0,0 +1,83 @@
+// <copyright file="AppConfig.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace StartTranscriptionByTimer
+{
+    using System;
+
+    using Connector.Constants;
+
+    public class AppConfig
+    {
+        private int messagesPerFunctionExecution = Constants.DefaultMessagesPerFunctionExecution;
+
+        private int filesPerTranscriptionJob = Constants.DefaultFilesPerTranscriptionJob;
+
+        private int initialPollingDelayInMinutes = Constants.DefaultInitialPollingDelayInMinutes;
+        private int retryLimit = Constants.DefaultRetryLimit;
+
+        public bool AddDiarization { get; set; }
+
+        public bool AddWordLevelTimestamps { get; set; }
+
+        public bool IsAzureGovDeployment { get; set; }
+
+        public bool IsByosEnabledSubscription { get; set; }
+
+        public int MessagesPerFunctionExecution
+        {
+            get => this.messagesPerFunctionExecution;
+            set => this.messagesPerFunctionExecution = Math.Clamp(value, 1, Constants.MaxMessagesPerFunctionExecution);
+        }
+
+        public int FilesPerTranscriptionJob
+        {
+            get => this.filesPerTranscriptionJob;
+            set => this.filesPerTranscriptionJob = Math.Clamp(value, 1, Constants.MaxFilesPerTranscriptionJob);
+        }
+
+        public int RetryLimit
+        {
+            get => this.retryLimit;
+            set => this.retryLimit = Math.Clamp(value, 1, Constants.MaxRetryLimit);
+        }
+
+        public int InitialPollingDelayInMinutes
+        {
+            get => this.initialPollingDelayInMinutes;
+            set => this.initialPollingDelayInMinutes = Math.Clamp(value, 2, Constants.MaxInitialPollingDelayInMinutes);
+        }
+
+        public int MaxPollingDelayInMinutes { get; set; } = Constants.DefaultMaxPollingDelayInMinutes;
```

Comment:
I believe you got a Math.Clamp here in the FetchTranscription AppConfig

Reply:
I saw `ClampInt` method being used from the `EnvironmentVariablesExtensions` class which is behaving same as `Math.Clamp`. So I updated the code accordingly in all the places. I could see it for both FetchTranscription & StartTranscription. Let me know if you feel something is missing or needs to be removed.

Reply:
no, what I meant is: 
in FetchTranscriptionConfig you got
```
        public int MaxPollingDelayInMinutes
        {
            get => this.maxPollingDelayInMinutes;
            set => this.maxPollingDelayInMinutes = Math.Clamp(value, 2, Constants.DefaultMaxPollingDelayInMinutes);
        }
```
and here you got:
```
public int MaxPollingDelayInMinutes { get; set; } = Constants.DefaultMaxPollingDelayInMinutes;
```

Reply:
Ah, nice catch, Thanks! I have updated it.